generator/templates/travis.yml.twig

```diff
@@ -204,6 +204,7 @@ before_script:
 
   - ./tests/travis/prepare.sh
   - ./tests/travis/setup_webserver.sh
+  - if [[ "$TRAVIS_PHP_VERSION" > 5.3* ]]; then ./tests/travis/install_phantomjs.sh; fi
```

Comment:
the `if [[ "$TRAVIS_PHP_VERSION" > 5.3* ]];` should prevent it from installing it for tests of Piwik 2.x, as UI tests are still executed on 5.3 there.